task_3/theater-gorm/pkg/data/theater.go

```diff
@@ -0,0 +1,646 @@
+package data
+
+import (
+	"fmt"
+
+	"gorm.io/gorm"
+)
+
+type Account struct {
```

Comment:
The same. 
Please create sperate files and structs for each model. As I explained for yours no-orm implementation